src/locales/es/translation.json

```diff
@@ -69,12 +69,12 @@
     "minutes": "minutos",
     "amount_hours": "{{amount}} horas",
     "amount_minutes": "{{amount}} minutos",
-    "accuracy": "{{accuracy}}% precisión",
+    "accuracy": "Completar al {{accuracy}}%",
```

Comment:
I'm curious about this one. `accuracy` is how precise is the time estimate for this category on HLTB

Reply:
This change seems to alter the meaning of the text

Reply:
Yup, I change it to make it a bit more understandable, precision could be interpreted in spanish as different ways (one example, precision of skills needed to completed in that time) , but it can be changed back to "precsión"